### PR TITLE
update thesis documentation

### PR DESCRIPTION
**Pull request by Brunner246**
- update `github-action` so that the docs are built once a week
- add diagrams for
    - git strategy 
    - time scheduling (project management)
- add chapters for thesis documentation

### CODE REVIEW

**Comment by Brunner246** [comment]
action is fixed and is now just executed on push on main.

File: Writerside/topics/Git-Strategy.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
+# Git Strategy
+
+GitFlow is a workflow that defines a strict branching model designed around the project release. It assigns very
```
Comment (munterfi) [inline Writerside/topics/Git-Strategy.md:3]: Are we sure we want to work with gitflow? I feel like a trunc-based approach would be leaner: No develop branch, merge short-living feature branches directly into master. Lets discuss at the workshop.

**Review by munterfi — APPROVED** [review]
Nice, thanks!

File: Writerside/topics/Git-Strategy.md (hunk 1 above)
Comment (Brunner246) [reply Writerside/topics/Git-Strategy.md:3]: i just added that sample from the mermaid docs. Lets build our own process in a workshop.

**Pull request merged**